### ISSUE TITLE
Изменение количества мест в рейсе

### ISSUE THREAD

**Issue by 3dlfive** [3dlfive/stepbooking#42]
Где мы это будем делать и проверять?
В теории можем сделать проверку в консоли . 
Я за вариант в консоли когда юзер вводит количество тогда проверку делаем. ну и после создания брони изменить количество свободных мест в самолёте и так же это сделать с удалением бронирования и изменения билетов...

Такой подход сделать контроллеры разнесенными и не зависимыми друг от друга.

Кто что думает

**Comment by khatsubavladimir** [comment]
ну это самое логичное решение, рассчитывалось же что все проверки делаются при вводе, я на вводном спрашивал - сказали можно либо в контроллерах либо в консоли, ну а поскольку консоль раньше идёт - лучше ошибку перехватить сразу там

**Comment by 3dlfive** [comment]
@khatsubavladimir можешь допилить это в консоли или помочь @Munkov разобраться

**Comment by khatsubavladimir** [comment]
боюсь что я сам не понимаю эту структуру консоли, со всеми этими меню, но вечером попробую разобраться

**Comment by 3dlfive** [comment]
> боюсь что я сам не понимаю эту структуру консоли, со всеми этими меню, но вечером попробую разобраться

class CreateBooking там идёт создание букинга,  там необходимо добавить проверку + проблема что нет документации как пользоваться твоим контроллером, Там нужно сделать чтобы загрузилась полёты с файла , а не  с каждым новым запуском генерировался новый файл
+ можно сделать Ексепшен который будет выбрасываться когда 
`aticket` будет больше чем количество свободніх мест +
потом нужно изменить в Флайтсах у того рейса количество свободных мест и перезаписать файл бд с флайтсами.

+ похожая тема с классом 
class CancelBooking при удалении необходимо взять количество мест с того рейса и добавить места в Флайтс, тоесть после строки:
    `    controller.dropByUID(in.nextLine());`

**Comment by khatsubavladimir** [comment]
добавил пул реквестом описание методов по примеру с букинг контроллером

**Comment by 3dlfive** [comment]
Я так понимаю и Флайтса должно быть максимальное количество или как происходит проверка

**Comment by khatsubavladimir** [comment]
нет, его нет, но его нельзя превысить в любом случае, все букинги только забирают места, и возвращают при удалении ровно столько же

**Comment by khatsubavladimir** [comment]
по сути максимальное количество мест рейса это то с которым он создан

**Comment by 3dlfive** [comment]
> по сути максимальное количество мест рейса это то с которым он создан

я про то что если в сгенерированіх полетах 
250 мест,
и мы удаляем букинг прото, оно долнж добавить место в рейс, а как оно узнает до какого максимального числа можно добавлять, ну или написать ошибку что места все свободные

**Comment by 3dlfive** [comment]
Изменения должны быть в класе CreateBooking и CancelBooking
